rust/android/Dockerfile

```diff
@@ -7,11 +7,8 @@ RUN apt-get update \
  && apt-get upgrade -y \
  && apt-get install -y --no-install-recommends --no-install-suggests \
             ca-certificates \
-            make \
             curl \
             wget \
-            gcc \
-            make \
             automake \
             build-essential \
```

Comment:
Small point, but I'd still prefer to see this line broken out into its parts: ennumerate the deps we need, dump anything we don't. This avoids the 'meta-package' pattern where we're getting a bundle of stuff, but reading it we don't know what it is, and if there's anything we don't need we'll have a smaller image, and so on.